classes/notification/Notification.php

```diff
@@ -199,4 +199,19 @@ public function scopeWithRead(Builder $query, bool $read)
             fn ($q) => $q->whereNull('date_read')
         );
     }
+
+    public static function getUnreadNotificationsCount(?int $userId): int
```

Comment:
In general it's better to avoid static methods that depend on global state, they are harder to test, but this whole notification stuff is going to be refactored one day, so nothing to do here, just a note.